### DISCUSSION TITLE
What happened to the Junit xml reports???

### DISCUSSION THREAD

**Discussion by anthonyemerson** [sbt/sbt#6320] [Q&A] [answered]
I am trying to update from sbt 1.2.8 up to 1.4.7 but when I try to run my tests in Jenkins all the Junit style xml files are missing. The docs still say the Junit Plugin is on by default, but just in case I tried setting `val myProject = (project in file(".")).enablePlugins(plugins.JUnitXmlReportPlugin)` as the docs suggest using `disablePlugins` to turn it off I just assumed the opposite would work. Unfortunately it didn't, nothing I can find anywhere suggests what happened to these reports, I cannot find them anywhere in the workspace, but without them my entire Jenkins pipeline would shatter.

Long story short I am hoping there is a toggle I am missing somewhere that just hasn't made it into the docs. Otherwise I will need to file a bug report.

**Comment by eed3si9n** [comment, marked as answer]
I think it creates a different name starting with `TEST-` and to get the old behavior you have to pass in `-Dsbt.testing.legacyreport=true` https://github.com/sbt/sbt/pull/5350

**Reply by anthonyemerson** [reply]
So for a while I was seeing both `TEST-` and the legacy format. But now the folder is completely empty, I can't find a single xml file in all of the workspace let alone the target directory

**Reply by eed3si9n** [reply]
I guess it's a bug then?

**Reply by anthonyemerson** [reply]
Sounds good I'll move this to issues then.

**Reply by anthonyemerson** [reply]
Pay no mind this was user error, in sbt 1.2.8 both reports were generated so we had a script deleting all the new style reports
